guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -48,8 +48,14 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
-. Enter the details for how you want the registered host to be configured.
+. Enter the details for how you want the registered host to be configured. Note:
+* If you select a host group from the *Host Group* list, the *Operating system*
 ifdef::katello,satellite,orcharhino[]
+, *Activation Keys*
+endif::[]
+and *Lifecycle environment* fields inherit their values from the host group.
```

Comment:
The fields are inherited from the host group but not preselected. Is that intentional? On Foreman 3.13:

![image](https://github.com/user-attachments/assets/1e5381d0-f9d8-42d8-8a2a-8df125e00b1e)

@thorben-d @mariaaga